mocks/data/items.ts

```diff
@@ -35,7 +37,7 @@ const items: Record<string, TestItemResult> = {
       actionType: "New",
     },
   },
-  ["VA"]: {
+  [WEIRD_ID]: {
```

Comment:
There is a certain check that wants to make sure an ID is just a wrong format for an item but it exist with in our system hence "weird"